TAF_EasyRest/TestFramework/PageComponents/AdministratorPanelComponents/WaitingForConfirmListPageComponent.cs

```diff
@@ -0,0 +1,25 @@
+﻿
+namespace TestFramework.PageComponents.AdministratorPanelComponents
+{
+    public class WaitingForConfirmListPageComponent
+    {
+        public IWebDriver driver { get; }
```

Comment:
a driver could not be public(!!!), it breaks the encapsulation principle, please change